packages/reaction-core/server/methods/orders.js

```diff
@@ -56,24 +56,25 @@ Meteor.methods({
     check(order, Object);
     check(shipment, Object);
     check(packed, Boolean);
-    this.unblock();
 
     if (order) {
-      ReactionCore.Collections.Orders.update({
-        "_id": order._id,
-        "shipping._id": shipment._id
-      }, {
-        $set: {
-          "shipping.$.packed": packed
-        }
-      });
-
       // Set the status of the items as shipped
       const itemIds = shipment.items.map((item) => {
         return item._id;
       });
 
-      Meteor.call("workflow/pushItemWorkflow", "coreOrderItemWorkflow/packed", order, itemIds);
+      const result = Meteor.call("workflow/pushItemWorkflow", "coreOrderItemWorkflow/packed", order, itemIds);
```

Comment:
This is exactly what we need. With callback we could don't get a result, because method call will be async with callback. We need 100% to have a method call result, that's why we should wait till it will be computed.